### ISSUE TITLE
[Bug]: [P1] Common accessibility issue - Focus order not coherent in components variants

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#588]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.16.0

### Describe the issue

Vocalization order is not coherent in main components variants: customize settings are vocalized before the title and main content.

### Expected Behavior

Check focus order : the first focused element should be the title, then main content and finally customization settings

**Comment by pya35** [comment]
fixed
